modules/sdk-core/src/account-lib/mpc/tss/ecdsa/types.ts

```diff
@@ -197,58 +191,90 @@ export interface GShare {
   mu: string;
   beta: string;
   nu: string;
-}
-export interface SignConvert {
-  xShare?: XShareWithNtilde; // XShare of the current participant
+};
+
+export type SignConvert = {
+  xShare?: XShareWithChallenges; // XShare of the current participant
   yShare?: YShare; // YShare corresponding to the other participant
   kShare?: KShare; // KShare received from the other participant
   bShare?: BShare; // Private Beta share of the participant
   muShare?: MUShare; // muShare received from the other participant
   aShare?: AShare;
   wShare?: WShare;
-}
+};
+
+export type SignConvertStep1 = {
+  xShare: XShareWithChallenges; // XShare of the current participant
+  yShare: YShare; // YShare corresponding to the other participant
+  kShare: KShare; // share to be modified and sent to other participant
+};
+
+export type SignConvertStep1Response = {
+  bShare: BShare; // participant share
```

Comment:
Kind of an ultra-nit, but for people not familiar with the ECDSA TSS signing flow, they might not immediately understand that "participant share" == "private share not to be given to anyone else"

It is called out in the type just above, so I think it's fine

Reply:
Addressing in https://github.com/BitGo/BitGoJS/pull/3602